modules/openy_features/openy_node/modules/openy_node_alert/openy_node_alert.install

```diff
@@ -164,4 +164,14 @@ function openy_node_alert_update_8006() {
       $config_updater->update($config, $config_name, $param);
     }
   }
+
+  // Import new configuration.
```

Comment:
@hamrant we have to add this as a hotfix to the dev branch since it should be executed and applied for all Open Y installation who already uses 8.1.11 version(these features were introduced in this version). 
As a solution let's leave this code here and create a new openy hook update(right in the profile) to perform it earlier than other hook updates.

Reply:
cc @podarok

Reply:
@ddrozdik fixed in this PR - https://github.com/ymcatwincities/openy/pull/1111